### PR TITLE
Fix is_installed test on Windows

### PR DESCRIPTION
**Pull request by Torxed**
`which` isn't a windows command-line tool. `where` however is and works in a similar manner. Added a fix for this, and now it runs on windows.

### CODE REVIEW

**Comment by Torxed** [comment]
I should note to any windows users who happens to stumble across this. `qcachegrind` needs to be in your PATH environment variable. The easiest way to fix this is to place your `qcachegrind` catalogue under `C:\Program Files (x86)\` and add `C:\Program Files (x86)\qcachegrind` to the PATH variable.

However this is a no-brainer, but pyprof2calltree assumes `kcachegrind` or `qcachegrind` is in the PATH.

File: pyprof2calltree.py [hunk 1]
```diff
@@ -137,7 +137,14 @@ def pstats2entries(data):
 def is_installed(prog):
     """Return whether or not a given executable is installed on the machine."""
     devnull = open(os.devnull, 'w')
-    retcode = subprocess.call(['which', prog], stdout=devnull)
+    # if where or which isn't located, a "FileNotFound" error will occur.
+    # in this case, we should catch it and return a null value so that:
+    #   if available_cmd is None:
+    # can trigger later on. But this is a quick fix to get it to work on Windows.
```
Comment (pwaller) [inline pyprof2calltree.py]: I agree with your analysis. Please remove the comment and instead put in a `try`/`except FileNotFound` block.

**Comment by Torxed** [comment]
@pwaller As per request, the `try` block has been created.

**Comment by pwaller** [comment]
Released in 1.4.3: https://pypi.python.org/pypi/pyprof2calltree/1.4.3

**Pull request merged**